src/App.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import './App.css';
-import { Home, About, Contact, Navigation } from './components';
+import { Home, About, Contact, Navigation } from './components'; 
+import {Route} from 'react-router-dom';
 
 const App = () => (
-  <div>
+  <div>  
+    <Route exact path = {`/`} component = {Home} ></Route>
```

Comment:
Curly braces around a url path imply that you're using javascript to create that url path. Because you're not doing that in this case, you don't need the curly braces.